wireapp/wire-android
# Feature Request: Shortcuts from home screen (New message, New Group Conversation, Share Photo)

## What's new in this PR?

![shortcuts_wire](https://user-images.githubusercontent.com/57259490/71247099-5e823400-2318-11ea-88c9-92d88e9ecda5.jpg)

This a feature implemented for API 25 > where you're able to quickly access features within the app from the home-screen. This is to improve UX and perform quick actions that may be a common use-case for users. 

### Testing

1.  Install the app 
2. Hold down on icon (you shouldn't be able to see shortcuts)
3. Login to Wire 
4. Go to homescreen and long press on Wire icon 
5. You should be able to see shortcuts 
6. Perform action

### Notes

This has been reviewed by Design and QA. This will only be allowed for internal builds until product has approved in the New Year. 

#### APK
[Download build #688](http://10.10.124.11:8080/job/Pull%20Request%20Builder/688/artifact/build/artifact/wire-dev-PR2501-688.apk)
[Download build #696](http://10.10.124.11:8080/job/Pull%20Request%20Builder/696/artifact/build/artifact/wire-dev-PR2501-696.apk)
[Download build #770](http://10.10.124.11:8080/job/Pull%20Request%20Builder/770/artifact/build/artifact/wire-dev-PR2501-770.apk)
[Download build #804](http://10.10.124.11:8080/job/Pull%20Request%20Builder/804/artifact/build/artifact/wire-dev-PR2501-804.apk)
[Download build #813](http://10.10.124.11:8080/job/Pull%20Request%20Builder/813/artifact/build/artifact/wire-dev-PR2501-813.apk)
[Download build #814](http://10.10.124.11:8080/job/Pull%20Request%20Builder/814/artifact/build/artifact/wire-dev-PR2501-814.apk)
